README.md

```diff
@@ -1,46 +1,98 @@
-# [gliff.ai](https://gliff.ai) ANNOTATE
+# gliff.ai ANNOTATE
 
-This repository contains the Open Source code for [gliff.ai](https://gliff.ai)'s ANNOTATE user interface.
-This user interface is just one part of [gliff.ai](https://gliff.ai)'s growing MLOps platform.
-ANNOTATE allows domain experts to annotate multidimensional images for developing imaging AI products.
-When used as part of [gliff.ai](https://gliff.ai)'s full platform ANNOTATE provides just one step in developing production-quality imaging AI whilst satisfying any relevant regulatory frameworks.
+![Latest Tag](https://img.shields.io/github/v/tag/gliff-ai/annotate?&label=latest_tag&style=flat-square&color=f2f2f2) ![Number of Open Issues](https://img.shields.io/github/issues/gliff-ai/annotate?style=flat-square&color=yellow) ![Number of Open Pull Requests](https://img.shields.io/github/issues-pr/gliff-ai/annotate?style=flat-square&color=yellow) ![Number of Contributors](https://img.shields.io/github/contributors/gliff-ai/annotate?style=flat-square&color=yellow) ![Repository Size](https://img.shields.io/github/repo-size/gliff-ai/annotate?style=flat-square&color=red) ![Repo License](https://img.shields.io/github/license/gliff-ai/annotate?color=0078FF&style=flat-square)
 
-User documentation will be available soon.
+👋 **Welcome in!** 👋
 
-Technical documentation will be available soon.
+This repository contains the Open Source code for [gliff.ai](https://gliff.ai)’s ANNOTATE product. 
 
-A preview build of the current `main` branch is [here](https://annotate.staging.gliff.app/)
+ANNOTATE aims to allow domain experts to easily and efficiently annotate multidimensional images - embedding their knowledge - for the purposes of developing imaging AI products. ANNOTATE is just one part of [gliff.ai](https://gliff.ai/)’s growing privacy-preserving MLOps (Machine Learning Operations) platform. When the full [gliff.ai platform](https://gliff.ai/software/) is used, ANNOTATE provides just one step in developing high-quality and auditable datasets that satisfy any relevant regulatory frameworks which enables our users to build world-changing and trustworthy AI models and products.
+
+A preview build of the current `main` branch is [here](https://annotate.staging.gliff.app/).
+
+✅ **We welcome contributions on this repository!** ✅
+
+## Demo-Preview
+
+_Coming soon._
```

Comment:
There is a staging demo at: https://annotate.staging.gliff.app/
@cooper667 shall we just link to this (same for other repos too)?

Reply:
yea, i think so, just explain it's built from the /examples folder on `main`? same for almost every repo